editors/code/src/lsp.ts

```diff
@@ -65,11 +65,6 @@ export class Lsp {
 				{ language: "r", pattern: "**/*.{r,R}" },
 				{ language: "r", pattern: "**/*.{rprofile,Rprofile}" },
 			],
-			synchronize: {
-				// Notify the server about file changes to R files contained in the workspace
-				fileEvents:
-					vscode.workspace.createFileSystemWatcher("**/*.[Rr]"),
-			},
```

Comment:
This causes the middleware to automatically start sending `DidChangeWatchedFiles` notifications to the Client

It was super confusing for me when I was registering the `air.toml` watched files notification because it looked like I had somehow done it wrong and was getting `.R` files too!